examples/browser/package.json

```diff
@@ -39,6 +40,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
+    "bunyan": "^1.8.10",
```

Comment:
I have to admit, I did not look into the complete changes yet, but I am positive, that the logging framework is not a dev dependency but a runtime one. In a nutshell; it should go the `dependencies`. Please correct me if I am wrong.